README.md

```diff
@@ -178,4 +208,49 @@ so for instance...
 
-Docker-compose will use the variables defined above, and put them under the `labels` section of a container (if a container needs it). The `labels` section is parsed by traefik to route all the HTTP calls to the correct containers.
+Docker-compose will use the variables defined above, and put them under the `labels` section of a container (if a container needs it). The `labels` section is parsed by traefik to route all the HTTP calls to the correct containers. In other words, this is where the values contained in the `.env` file are used to specify that the app replies on '`my.`appserver.test' instead of e.g. '`magic.`appserver.test'.
+
+### /etc/hosts
+
+This file which is located on the _HOST_ machine is used to add new [subdomains for the services](#subdomains-registered-on-traefik): e.g. add `pgadmin` to be able to go to `pgadmin.appserver.test` and use a visual tool for the databases. The important part is that this name of subdomain matches the one defined in the file `.env`
+
+# Troubleshooting / other
+
+- If you are stuck and something doesn't seem to work, make sure you don't have a mismatch between `/etc/hosts`, the URL you type in the browser, and the address that the system expects. See file `current-config.yml` for that: it is a file which is generated every time `make start` is launched. It contains the description of the desired state of the app.
+
+- You can add dev-tools to the array of services and troubleshoot docker using portainer (mentioned above). Make sure that you have added also the URL to the hostfile (`/etc/hosts`) and that it matches the variable defined in `.env`
+
+- If you remove services from array `ENABLED_SERVICES`, they are not stopped. This is not a problem in general, just don't be surprised if when you start (having the configuration `ENABLED_SERVICES=core:frontend:events`) and stop later (with the configuration `ENABLED_SERVICES=events`) you find a message `WARNING: found orphan containers`
+
+## How to run unit/integration tests when you start the system
+
+?? @serge1peshcoff
+
+## Default credentials
+
+From the [core readme](./core/README.md): there are test users. All users have `5ecr3t5ecr3t` as password.
+
+- `admin@example.com` - admin
+- `board@example.com` - board member of antenna
+- `member@example.com` - regular member of antenna
+- `not-confirmed@example.com` - member who is not confirmed
+- `password-reset@example.com` - member who requested a password reset
+- `suspended@example.com` - a suspended member
+
+You can use `5ecr3t` for a password reset token (for a member with email `password-reset@example.com`) and `5ecr3t` for a mail confirmation (for a member with email `not-confirmed@example.com`).
+
+**NOTICE** if you use the `--fast` mode of `start.sh`, then core does NOT provision in production mode, so you will be fast in bootstrapping but without users to play with.
+
+@serge1peshcoff please confirm
```

Comment:
@serge1peshcoff a question for you

Reply:
Also the admin is not provisioned at all?
Can I tell people to register normally if they need a user account, or would they be blocked by an email?
Is there a "quick" way for them to make the migration run even in production mode, or to provision any user without registration?